tests/e2e/functional_test.go

```diff
@@ -524,6 +541,56 @@ var _ = Describe("Functional e2e", Serial, func() {
 
 	})
 
+	It("Should update the child ISBService with an excluded field", func() {
+
+		// new ISBService spec
+		rawSpec, err := json.Marshal(ISBServiceSpecExcludedField)
+		Expect(err).ShouldNot(HaveOccurred())
+
+		updateISBServiceRolloutInK8S(isbServiceRolloutName, func(rollout apiv1.ISBServiceRollout) (apiv1.ISBServiceRollout, error) {
+			rollout.Spec.InterStepBufferService.Spec.Raw = rawSpec
+			return rollout, nil
+		})
+
+		if dataLossPrevention == "true" {
+
+			document("Verify that in-progress-strategy gets set to NoOp")
+			verifyInProgressStrategyISBService(Namespace, isbServiceRolloutName, apiv1.UpgradeStrategyNoOp)
+			verifyInProgressStrategy(pipelineRolloutName, apiv1.UpgradeStrategyNoOp)
+
+			if dataLossPrevention == "true" {
+				document("Verify that dependent Pipeline is not paused when an update to ISBService not requiring pause is made")
+				verifyPipelineStatusConsistently(Namespace, pipelineName, func(retrievedPipelineSpec numaflowv1.PipelineSpec, retrievedPipelineStatus numaflowv1.PipelineStatus) bool {
+					return retrievedPipelineStatus.Phase != numaflowv1.PipelinePhasePaused
+				})
+			}
```

Comment:
looks like `if dataLossPrevention == "true"` is in here twice

Reply:
Oops, fixing that now.